pkg/cmd/cluster/cluster.go

```diff
@@ -9,9 +9,10 @@ import (
 
 func NewCmdCluster(f *factory.Factory) *cobra.Command {
 	cmd := cobra.Command{
-		Use:  "cluster <command>",
-		Args: cobra.ArbitraryArgs,
-		Long: "View cluster information",
+		Use:   "cluster <command>",
+		Args:  cobra.ArbitraryArgs,
+		Long:  "View cluster information",
+		Short: "View cluster information",
```

Comment:
Oh hold on, the others are "manage" because it has things like build create/cancel/rebuild.
This will likely have more than just "view" as well so maybe?:
```suggestion
		Short: "Manage organization clusters",
```